styles/page_interfaces.less

```diff
@@ -25,6 +28,16 @@
 @page-interface-visited-color: #990099;
 @page-interface-active-color: #ff0000;
 
+/* WordCheck*/
+@preWC: #c2c2c2;
+@noWC: #ffb321;
+@WC: #97fc93;
```

Comment:
Was it intentional to change this from `#97fc9e`?  I don't have strong feelings about this subtle change but wanted to confirm you mean to make it.

Reply:
> Was it intentional to change this from `#97fc9e`? I don't have strong feelings about this subtle change but wanted to confirm you mean to make it.

Um, no, not intentional at all. Me fat-fingering it, I suppose (the 3 is above the e on the keyboard, and my finger gets them mixed up more often than I'd like 😁). I'll get that changed back.